pages/kavita+.mdx

```diff
@@ -63,4 +63,4 @@ From the Admin Dashboard -> Kavita+ tab, press the Activate button. This will pr
 ![Activate%20License](../public/kavita+/activate-license.png "Activate%20License")
 
 ## How to Unsubscribe
-If Kavita+ isn't bringing value to you, you can unsubscribe easily via the Kavita+ Tab. Click Manage which will open your Stripe portal and from there you can cancel your membership. You will receive an email stating your subscription has been canceled and when your Kavita+ membership lasts. You can use this Manage button at any time in the future to re-activate your subscription. If in doubt, please reach out to [majora2007](https://discord.com/invite/b52wT37kt7).
\ No newline at end of file
+If Kavita+ isn't bringing value to you, you can unsubscribe easily via the Kavita+ Tab. Click Manage which will open your Stripe portal and from there you can cancel your membership. You will receive an email stating your subscription has been canceled and when your Kavita+ membership lasts. You can use this Manage button at any time in the future to re-activate your subscription. If in doubt, please reach out to [majora2007](https://discord.com/invite/b52wT37kt7).
```

Comment:
Could we update this so that the unsubscribe is linked as well?

https://billing.stripe.com/p/login/28oaFRa3HdHWb5ecMM

Reply:
donez0rs